mdoc/src/main/scala/mdoc/internal/markdown/Mod.scala

```diff
@@ -38,7 +48,16 @@ object Mod {
       ToString,
       Nest
     )
+
+  def parametric: List[String => Option[Mod]] =
+    List(
+      s => Try(s.replace("width=", "").toInt).toOption.map(Width),
+      s => Try(s.replace("height=", "").toInt).toOption.map(Height)
+    )
+
   def unapply(string: String): Option[Mod] = {
-    all.find(_.toString.equalsIgnoreCase(string))
+    static.find(_.toString.equalsIgnoreCase(string)) orElse parametric
```

Comment:
nit: let's avoid non-symbolic infix operators
```suggestion
    static.find(_.toString.equalsIgnoreCase(string)).orElse {
      parametric.flatMap(check => check(string)).headOption
    }
```